package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "blob",
```

Comment:
You need to avoid using `blob` as the project name, it already exists at https://www.npmjs.com/package/blob .